sklearn/ensemble/tests/test_weight_boosting.py

```diff
@@ -320,6 +319,65 @@ def test_sample_weight_missing():
     assert_raises(ValueError, clf.fit, X, y_regr)
 
 
+def test_sample_weight_adaboost_regressor():
```

Comment:
Please move this to below test_sparse_classification so the diff is easier to read

Reply:
Fixed